db/migrate/20160718114501_update_existing_builtin_to_boolean1_in_roles.rb

```diff
@@ -0,0 +1,9 @@
+class UpdateExistingBuiltinToBoolean1InRoles < ActiveRecord::Migration
+  def up
+    roles = Role.where(:builtin => 2)
+    roles.each do |role|
+      role.builtin = 1
+      role.save!
```

Comment:
this should either use fake model or some AR method to skip validations to avoid problems in future, please see example in some [older migration](https://github.com/theforeman/foreman/blob/develop/db/migrate/20140910153654_move_host_nics_to_interfaces.rb#L9)